docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -0,0 +1,19 @@
+---
+id: pillars-fabric-components
+title: Fabric Components
+---
+
+This section contains a high-level introduction to Fabric components. It provides enough context to understand when a Fabric component is needed and how it roughly works.
+It points to the [Renderer](https://reactnative.dev/architecture/fabric-renderer) section of the [Architecture](https://reactnative.dev/architecture/overview) tab for a deep dive into the technical details.
```

Comment:
```suggestion
It points to the [Renderer](/architecture/fabric-renderer) section of the [Architecture](/architecture/overview) tab for a deep dive into the technical details.
```

Also we can improve those links.

Reply:
This will change as soon as we merge the actual guide! 👍